### ISSUE TITLE
[Enhancement]: Improved access to WooCommerce logging

### ISSUE THREAD

**Issue by terriann** [woothemes/woocommerce#31823]
### Describe the solution you'd like

When a user has logs displayed in `/wp-admin/admin.php?page=wc-status&tab=logs` they need to query the database to get the additional details stored in the `context` field in the database schema. Not all WooCommerce users may have access to the database or the knowledge on how to extract the additional information.

Some improvements that would help make this information more accessible for both developers and non-developers:

1) The HTML table displayed in that page should include the primary key (`log_id`) for the log entry to make it easier to query the details for a specific entry using a SQL query like: `SELECT * FROM wp_woocommerce_log WHERE log_id = 123;`

2) The information in the `context` field could be displayed in the UI, possibly using the ["show more details" button which is included in the existing row actions in the `WP_List_Table class`](https://github.com/WordPress/WordPress/blob/ca9b189085c42f7f208fc174c20cc1fce81c69df/wp-admin/includes/class-wp-list-table.php#L572) used on that page.

3) In combination with the above suggestion to display the `log_id` in the table, development teams with access to the WP-CLI might find it useful to query that information using commands like `wp wc log list` and `wp wc log get 123`

In both 2 and 3 above, displaying the unserialized contents of the `context` field would be more useful than having to deserialize the contents after running a SQL query.

### Describe alternatives you've considered

The current workaround is to query the database directly with limit queries like `SELECT * FROM wp_woocommerce_log ORDER BY log_id DESC LIMIT 3;` to match records displayed in the UI with records in the database.

### Additional context

The specific example we'd looked at, log entries had the message "Error saving order" in the UI. Upon further investigation, using the ID of the impacted order from the `context` database field, we were able to see that the issue had been corrected.

ex. `2022-01-22 25:34:55 |   500 | class-wc-order | Error saving order. | a:2:{s:5:"order";O:44:"Automattic\WooCommerce\Admin\Overrides\Order":1:{s:5:" * id";i:123456;}`

**Comment by johnregan3** [comment]
I'll take this one on once #32081 is merged.

**Comment by johnregan3** [comment]
@barryhughes Tagging just so there is some visibility on this PR. Feel free to reassign as needed.

**Comment by rrennick** [comment]
cc @coreymckrill

**Comment by coreymckrill** [comment]
#41936 makes it so that, going forward, context data is saved as a JSON string instead of serialized. It also adds a way to view the context data from the logs list table. I think this basically addresses the need to view the context data expressed in this issue, even though it doesn't add any wp-cli commands. I'm going to go ahead and mark this as completed, but we can open another issue specific to wp-cli commands if there is still a need.
